From bc356914890dd6ee425a1a7f2308759fbf918e95 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 9 May 2024 11:24:26 +1100
Subject: [PATCH 5/5] try harder to set up s3_server fixture

it only happens once per session, so we may as well take the time
---
 python/pyarrow/tests/conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/pyarrow/tests/conftest.py b/python/pyarrow/tests/conftest.py
index a5941e8c8..e35ac92e9 100644
--- a/python/pyarrow/tests/conftest.py
+++ b/python/pyarrow/tests/conftest.py
@@ -193,7 +193,7 @@ def retry(attempts=3, delay=1.0, max_delay=None, backoff=1):
 
 @pytest.fixture(scope='session')
 def s3_server(s3_connection, tmpdir_factory):
-    @retry(attempts=5, delay=0.1, backoff=2)
+    @retry(attempts=5, delay=1, backoff=2)
     def minio_server_health_check(address):
         resp = urllib.request.urlopen(f"http://{address}/minio/health/cluster")
         assert resp.getcode() == 200
